examples/power_distribution.py

```diff
@@ -82,10 +82,10 @@ async def run(self) -> None:
             _logger.debug("Avg power %d", avg_power)
             if avg_power > 30000:
                 # Charge
-                power_to_set: int = 10000
+                power_to_set: float = 10000.0
```

Comment:
Even when not related to this PR, I recommend *declaring* the type outside of the `if` here to make it more clear:

```py
            power_to_set: float
            if avg_power > 30000:
                # Charge
                power_to_set = 10000.0
```

Reply:
Updated